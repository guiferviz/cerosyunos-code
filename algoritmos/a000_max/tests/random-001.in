4
43
654285996 799027278 974804266 296065994 1006262505 835455049 227021172 300613717 472590840 981120670 625491225 429499883 1158285105 2128360223 604133035 64524269 317713507 1343690809 271133419 1039024689 1493531714 303346194 1230251692 1671767926 1920870543 384509727 1701305456 1005661294 1330242909 1547171 1115667419 39643997 2121087343 978030569 1704108206 642125421 1009871181 1098850293 223858944 1637045142 1633487900 637363624 1148566614
43
427617309 1276758800 895716253 1109149326 1366253137 681390543 2104475935 1727724028 915592501 577273388 1864129003 799593896 904296382 2123591120 1859694487 1592091678 1977613182 1522999082 856952475 1050559651 344952499 981366268 322552122 479513276 1414425236 1462685361 1070698509 78699427 1640976901 603657757 1652875156 1359204541 1130689507 1449631665 891359644 1991911247 1121309968 1878421104 1432234591 1077162839 1999028551 334238127 269203004
43
1345834347 825076373 1921448502 131441627 2119200224 843977438 464951458 192175700 377217735 1548706637 1269973488 916717136 29809402 1108646044 838534958 1703342601 901177185 672415624 1531470364 439363595 978969127 22996075 301212778 1254886352 890821981 128741197 1602229654 9340196 1960060968 1036326296 1475907251 967320725 1927554634 1877649863 656556576 1364561427 550793679 1461501307 804727879 905149971 2115379377 1129461893 1046879440
43
1338318630 610890441 598842944 1115376457 162406567 710146693 365708539 2010635098 2102360373 1760314521 94985961 1067574799 65291593 551998083 1965210743 739233213 15405816 165186150 38423285 1936770680 325731091 2009653942 100217955 720963026 1148895552 838664881 1224911901 75196634 1510611503 388140335 1587157976 1657918813 91104539 451981685 1415713765 1012146228 1125525805 1610014405 276736347 313926930 648106267 2040644030 1143234308
