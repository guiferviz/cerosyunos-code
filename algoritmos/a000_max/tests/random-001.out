2128360223
2123591120
2119200224
2102360373
